tests/sentry/grouping/ingest/test_seer.py

```diff
@@ -261,3 +262,55 @@ def test_returns_no_grouphash_and_empty_metadata_if_no_similar_group_found(self)
                 expected_metadata,
                 None,
             )
+
+    @patch("sentry.seer.similarity.utils.metrics")
+    def test_too_many_only_system_frames(self, mock_metrics: Mock) -> None:
```

Comment:
Do we have a test that tracks what happens if you have N exceptions and the sum of their frames is over MAX_FRAMES?

Do we have a test that handles multiple exceptions but only 1 frame is in-app?

Reply:
Thanks for the suggestion! I've added a test for the first case.
Regarding the second case, if we have app data, it's used instead of system data (see [here](https://github.com/getsentry/sentry/blob/f8810ea5dfcd50d1a6a790b39be0b84c8dc1de8b/src/sentry/seer/similarity/utils.py#L169-L170)) and we have a test for it [here](https://github.com/getsentry/sentry/blob/f8810ea5dfcd50d1a6a790b39be0b84c8dc1de8b/tests/sentry/seer/similarity/test_utils.py#L698)